templates/endpoints_v2/README.ipynb

```diff
@@ -228,6 +228,13 @@
     "* [Learn how to leverage different configurations that can optimize the latency and throughput of your models](examples/OptimizeModels.ipynb)\n",
     "* [Learn how to fully configure your deployment including auto-scaling, optimization parameters and tensor-parallelism](examples/AdvancedModelConfigs.ipynb)\n",
     "\n",
+    "## Cookbooks\n",
```

Comment:
can you move the more guides to under cookbooks? All of them are cookbooks now. 

Also move the RayLLM explanations to something under maybe "What is RayLLM?". The flow is awkward right now.

Reply:
Done